routes/api.php

```diff
@@ -957,21 +957,20 @@
                     Api\SettingsController::class,
                     'downloadBackup'
                 ]
-            )->name('api.settings.backups.download');
-
-        }); 
-        
-        Route::resource('settings', 
-        Api\SettingsController::class,
-        ['names' => [
-                'show' => 'api.settings.show',
-                'update' => 'api.settings.update',
-                'store' => 'api.settings.store',
-            ],
-        'except' => ['create', 'edit', 'index', 'destroy'],
-        'parameters' => ['setting' => 'setting_id'],
-        ]
-        ); // end settings API
+            )->name('api.settings.backups.download');  
```

Comment:
I don't think we need this? We don't allow updating most of the settings via API.